ng2-components/ng2-alfresco-core/src/components/info-drawer/info-drawer.component.ts

```diff
@@ -36,10 +36,18 @@ export class InfoDrawerComponent {
     @Input()
     title: string|null = null;
 
+    @Output()
+    tabChangeEvent: EventEmitter<any> =new EventEmitter<any>();
```

Comment:
can you call it 
tabChange and add a space after the = 
like
tabChange: EventEmitter<any> = new EventEmitter<any>();